### DISCUSSION TITLE
Thank you for this package!

### DISCUSSION THREAD

**Discussion by justwheel** [tajmone/ST4-Asciidoctor#38] [General]
Hi, just wanted to share my gratitude and thanks for this package! I installed it and immediately had AsciiDoc syntax highlighting. I work with AsciiDoc frequently so this was super useful for me. Thanks!

**Comment by tajmone** [comment]
I'm glad you like it, but there are still some problems to fix.

1. The AsciiDoc syntax is quite complex to handle, and this package has a long way to go in that respect.
2. The package doesn't self update.

As for point (2) the problem is that ST package manager can't really handle properly Git repositories installed without going through the packages repository. I've been tinkering with the idea of creating a dedicated package manager to install packages from Git repositories, and update them automatically. But I'm not quite sure I'll have the time for that (also, not quite happy about the direction ST is taking).

For the time being, you'll have to manually update the package via Git, or setup some script that handles that periodically (e.g. at startup).
